### PR TITLE
Fix: handling errors in app

### PR DESCRIPTION
**Pull request by L0wry**

### CODE REVIEW

**Comment by kaliabadi** [comment]
Looks like you might need to upgrade react-native-web

File: package.json [hunk 1]
```diff
@@ -65,8 +65,8 @@
     "webpack-merge": "^4.1.1"
   },
   "peerDependencies": {
-    "react": ">= 16.0.0-alpha.6 <= 16.0.0",
-    "react-native": ">= 0.44.1 <= 0.50"
+    "react": "16.2.0",
+    "react-native": ">= 0.50.1"
```
Comment (rjanjua) [inline package.json]: only change the upper limit, rather than tie it into a specific version

Comment (rjanjua) [inline package.json]: only change the upper limit

File: package.json [hunk 2]
```diff
@@ -65,8 +65,8 @@
     "webpack-merge": "^4.1.1"
   },
   "peerDependencies": {
-    "react": ">= 16.0.0-alpha.6 <= 16.0.0",
-    "react-native": ">= 0.44.1 <= 0.50"
+    "react": "16.2.0",
```
Comment (rjanjua) [inline package.json]: only change the upper limit rather than tying it in to a specific version

File: package.json (hunk 1 above)
Comment (L0wry) [reply package.json]: 😢

**Pull request merged**